go/vt/tableacl/tableacl.go

```diff
@@ -39,72 +68,82 @@ func Init(configFile string) {
 
 // InitFromBytes inits table ACLs from a byte array.
 func InitFromBytes(config []byte) (err error) {
-	tableAcl, err = load(config)
-	return
+	return load(config)
 }
 
 // load loads configurations from a JSON byte array
 //
 // Sample configuration
 // []byte (`{
-//	<tableRegexPattern1>: {"READER": "*", "WRITER": "<u2>,<u4>...","ADMIN": "<u5>"},
-//	<tableRegexPattern2>: {"ADMIN": "<u5>"}
+//	<table name or table name prefix>: {"READER": "*", "WRITER": "<u2>,<u4>...","ADMIN": "<u5>"},
+//	<table name or table name prefix>: {"ADMIN": "<u5>"}
 //}`)
-func load(config []byte) (map[*regexp.Regexp]map[Role]acl.ACL, error) {
-	var contents map[string]map[string]string
-	err := json.Unmarshal(config, &contents)
-	if err != nil {
-		return nil, err
+func load(data []byte) error {
+	config := &tableaclpb.Config{}
+	if err := proto.Unmarshal(data, config); err != nil {
+		return err
 	}
-	tableAcl := make(map[*regexp.Regexp]map[Role]acl.ACL)
-	for tblPattern, accessMap := range contents {
-		re, err := regexp.Compile(tblPattern)
+	var entries aclEntries
+	for _, group := range config.TableGroups {
+		readers, err := newACL(strings.Split(group.Readers, ","))
```

Comment:
OK now reading this, let's use repeated fields in the proto, and not split here, you'll get a []string directly.